README.md

```diff
@@ -15,7 +15,7 @@ Teams are not left on their own to solve the challenges. Coaches work with each
 ### Data & AI
 - [MLOps on GCP](./hacks/mlops-on-gcp/README.md)
   > We will be implementing the full lifecycle of an ML project. We’ll provide you with a sample code base and you’ll work on automating continuous integration (CI), continuous delivery (CD), and continuous training (CT) for a machine learning (ML) system. 
-- [Data Analytics for Retail](./hacks/retail-analytics/README.md)
+- [Real-time analytics with CDC](./hacks/retail-analytics/README.md)
```

Comment:
I don't like the acronym here... at the very least, make mention of Cloud Data Capture in the blurb.